src/components/WhaleSightingDetail/WhaleSightingDetail.tsx

```diff
@@ -1,11 +1,18 @@
 import "./WhaleSightingDetail.scss"
+import { SightingDetail } from "../SightingDetail/SightingDetail";
+import { useParams } from "react-router-dom";
 
 export const WhaleSightingDetail: React.FunctionComponent = () => {
+    const { id } = useParams();
+
+    if (id === undefined) {
+        return <h1> No Whale Sighting Found </h1>
+    }
     return <main>
-        <h1 className="main-header">Whale Sighting Detail Page</h1>
+        <section className="header-section"><h1 className="main-header">Whale Sighting Detail Page</h1></section>
```

Comment:
Can we separate this onto multiple lines to make it easier to read?
```suggestion
        <section className="header-section">
            <h1 className="main-header">Whale Sighting Detail Page</h1>
        </section>
```